pkg/k8s/client/client_test.go

```diff
@@ -11,12 +11,14 @@ import (
 	"time"
 
 	. "github.com/cilium/checkmate"
+	"github.com/cilium/hive/hivetest"
 	"github.com/spf13/pflag"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/cilium/hive/cell"
```

Comment:
General nit/question (applies to all files, but here is a good example where the discrepancy shows): For other packages under the cilium GitHub organization (e.g. cilium/dns) we don't introduce a new group in the import list. Here it's even inconsistent between `cilium/hive/cell` and `cilium/hive/hivetest`.

IMO, the `github.com/cilium/hive/cell` import should belong to the third-party group as all other dependencies in `vendor/`.

Reply:
For context, here's the ordering we decided on a few years ago - https://github.com/cilium/cilium/pull/18357

I'm surprised this isn't enforced by any CI workflow to be honest

Reply:
https://github.com/cilium/cilium/pull/18357#discussion_r777612575 I guess says it best, I really don't want to spend time manually fixing up stuff

Reply:
Yeah, so it seems that our tooling is insufficient :disappointed: 

I'm pretty sure there are probably already a bunch of already broken import statements in the tree because of that.

However, I still feel uneasy about merging this PR as is, since it objectively does introduce more inconsistencies, even if our linters don't complain about it. I did a quick test with `gofumpt` and it also does not seem to pick up the broken imports :/

Reply:
Yeah, fair enough. I'll hack something together with duct tape and sed to fix this up :safety_vest: